### PR TITLE
Add pwd and counters to mfu dump format

### PR DESCRIPTION
**Pull request by bogiton**

### CODE REVIEW

File: client/cmdhfmfu.h [hunk 1]
```diff
@@ -21,7 +21,8 @@ typedef struct {
 	uint8_t	pack[2];
 	uint8_t tbo1[1];
 	uint8_t	signature[32];	
-	//uint8_t	counter[3];
+	uint8_t	counter[3][3];
```
Comment (iceman1001) [inline client/cmdhfmfu.h]: Isn't counters 24bits?  uint32_t[3] ?
Or keep it even simpler,   Counter1,  Counter2, Counter3  ?

I guess this messes up old dumps though when reading them.   One of the main reasons I didn't fiddle with this part ...

File: client/cmdhfmfu.h [hunk 2]
```diff
@@ -21,7 +21,8 @@ typedef struct {
...
+	uint8_t pwd[4];
```
Comment (iceman1001) [inline client/cmdhfmfu.h]: And authentication key could be  4bytes or 16 (triple des  ULC)

File: client/cmdhfmfu.h (hunk 1 above)
Comment (bogiton) [reply client/cmdhfmfu.h]: Yes, 3 counters of 3 bytes each. Thus the double array.

File: client/cmdhfmfu.h (hunk 2 above)
Comment (bogiton) [reply client/cmdhfmfu.h]: You are right. I was only thinking of ev1...

**Comment by bogiton** [comment]
I actually changed the mfu_dump_t struct because of the json output. I didn't know that other locations are depending on that.
Maybe I should close this one and make separate ones.

**Pull request closed without merging**